interfaces/IBF-dashboard/src/app/models/user/user-role.enum.ts

```diff
@@ -1,5 +1,7 @@
 export enum UserRole {
   Admin = 'admin',
+  SetTriggerUser = 'set-trigger-user',
```

Comment:
@jannisvisser I assumed the disaster manager is the set trigger user. All other users are guests, who cannot set triggers or (un)check EAPs. Is it possible that a disaster manager should not be allowed to set triggers?